### PR TITLE
Update library doophoenix v6 part3

### PR DESCRIPTION
**Pull request by manu1080**
Fixes: https://github.com/doofinder/doophoenix/issues/1753
Part 1: https://github.com/doofinder/js-doofinder/pull/312
Part 2: https://github.com/doofinder/js-doofinder/pull/312

Welcome wave to Part 3 of the migration to Doophenix V6. 

**The last Part.**

Summary
The problem arose during the hackathon of updating our JavaScript library that works with Doophenix V5 to version 6. Together with @davidmolinacano, we reviewed the new Doophenix API and managed to make all the endpoints work, but there is still a lot of work to be done. We created this issue to finish it.

**Update example files.**

Thank you all for your time! raised_hands

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -32,7 +32,9 @@
   "author": "Doofinder",
   "contributors": [
     "Carlos Escribano Rey <carlos@doofinder.com>",
-    "José Carlos Cuevas Albadalejo <josecarlos@doofinder.com>"
+    "José Carlos Cuevas Albadalejo <josecarlos@doofinder.com>",
+    "Manuel Irazabal <manuel.irazabal@doofinder.com>",
+    "David Molina <david@doofinder.com>"
```
Comment (davidmolinacano) [inline package.json:37]: Thank you for including me in the credits! :handshake:

File: playground/demo.js [hunk 2]
```diff
@@ -1,39 +1,111 @@
...
+
+  params = {
+    hashid,
+    session_id: sessionId,
+  }
+
+  stats.registerCheckout(params).then((result) => {
+    console.log('CHECKOUT: ', result);
+  });
+}
```
Comment (davidmolinacano) [inline playground/demo.js:111]: Try to add an empty line at the end of the file (Github complaints about this with a very ugly forbidden symbol)

File: playground/demo.js [hunk 3]
```diff
@@ -1,39 +1,111 @@
 function demo({ Client, Query, StatsClient }) {
-  const hashid = '7a6100c782c09126479e9270ecc619b3';
-  const sessionId = '123456';
-  const client = new Client({ server: 'eu1-search.doofinder.com' });
-  const request = new Query({ hashid });
+  const hashid = '';
+  const dfid =''
+  const secret = ''
+  const sessionId = '';
```
Comment (mrroot5) [inline playground/demo.js:5]: Why removed sample data?

**Review by davidmolinacano — APPROVED** [review]
Makes sense, great!! :medal_military:

File: playground/demo.js (hunk 3 above)
Comment (manu1080) [reply playground/demo.js:5]: esa data no hacia nada elusuario que lo use tendra que poner sus propios datos para que ande

Comment (CAxGu) [reply playground/demo.js:5]: En ese caso... quizá valdría la pena añadir una pequeña doc/domentario con un ejemplo de los valores que estas podrían tener?. Nosotros sabemos que pinta tiene un hashid, secret etc.. pero.. quizá quien utilice esto externamente no.

**Review by davidmolinacano — APPROVED** [review]
Excellent!! :clap:

File: playground/demo.js (hunk 3 above)
Comment (manu1080) [reply playground/demo.js:5]: Suena bien creo que lo a;adire al readme y un comentario en el archivo

File: playground/demo.js (hunk 2 above)
Comment (manu1080) [reply playground/demo.js:118]: Done

File: playground/demo.js (hunk 3 above)
Comment (manu1080) [reply playground/demo.js]: Done

**Review by CAxGu — APPROVED** [review]
:+1:

**Pull request merged**